elements/rh-navigation-primary/rh-navigation-primary.ts

```diff
@@ -410,11 +410,17 @@ export class RhNavigationPrimary extends LitElement {
   }
 
   async #openHamburger() {
+    if (!this._hamburger) {
+      await this.updateComplete;
+    }
     this._hamburger.open = true;
```

Comment:
perhaps unrelated to this PR, if so spin out to a separate issue

this (and below) should not imperatively set open, instead, use the lit template. set private state on the class, and have that toggle the open attr or prop via the template. ensure state is synchronized in the toggle listener.

we should not use the dom api to modify elements that lit manages, as lit can get temperamental about that.

Reply:
https://github.com/RedHat-UX/red-hat-design-system/issues/2408